lib/anyway/config.rb

```diff
@@ -198,6 +198,15 @@ def env_prefix(val = nil)
         end
       end
 
+
+      def loader_options(val = nil)
+        return (@loader_options = val) unless val.nil?
+
+        return @loader_options if instance_variable_defined?(:@loader_options)
+
+        @loader_options = {}
```

Comment:
I think, we also need to take care of inheritance (see other class-level configuration methods)